### ISSUE TITLE
TypeError: Cannot read property 'url' of undefined

### ISSUE THREAD

**Issue by LA1CH3** [contentful/contentful-export#29]
Steps to reproduce:
1. `npm i -g contentful-export` on Node 7.4.0.
2. Create a config.json like so (tokens replaced):
```
{
  "spaceId": "...",
  "managementToken": "...",
  "includeDrafts": "true",
  "previewToken": "...",
  "maxAllowedLimit": "5000"
}
``` 
3. run `contentful-export --config config.json`

It gets to "Getting content from source space" and then after about a minute I receive the following error message:

```
ERR! 
The destination space was not found. This can happen for multiple reasons:
- If you haven't yet, you should create your space manually.
- If your destination space is in another organization, and your user from the source space does not have access to it, you'll need to specify separate sourceManagementToken and destinationManagementToken
```

The readme says nothing regarding a destination space, and I'm not sure why that would even come up as this is simply exporting from a source space to a JSON file.

**Comment by Khaledgarbaya** [comment]
Hi @LA1CH3 Sorry for the message it is a bit confusing.
maxAllowedLimit max is 1000 which means how many record per page you can get. Also, you no longer need the `previewToken` (Sorry about that I just updated the token).
Ok now for the main reason for this error could be an error in the managementToken or the spaceId.
Make sure it is the management token you got from https://www.contentful.com/developers/docs/references/authentication/

**Comment by LA1CH3** [comment]
Thanks for the quick response @Khaledgarbaya . I have changed my config object to this:

```
{
  "spaceId": "...",
  "managementToken": "...",
  "includeDrafts": "true",
  "maxAllowedLimit": "1000"
}
```

And it now works. I also was behind a corporate proxy- I switched to a regular network and it worked, so that was probably the original issue. However, I'm running into something different now. 

I get the exported object back, but I find that my `entries` array from the export is completely empty. Any idea? Assets, Content Types, and the other expected fields are showing values.
